package.json

```diff
@@ -26,8 +26,9 @@
     "clean": "rimraf ./dist",
     "test": "npm run test:version && npm run lint && npm run test:cov",
     "test:version": "node scripts/check-version.js",
-    "test:cov": "c8 npm run test:test",
+    "test:cov": "c8 npm run test:ci",
```

Comment:
`test:cov` is also meant to be used locally, so I suggest we don't replace its functionality. If we need a separate script specifically for CI, let's add it

Reply:
Makes sense, will make the change